examples/kubernetes-1ppc/configmaps.yml

```diff
@@ -0,0 +1,39 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: st2
+data:
+  ST2_USER: admin
+  ST2_PASSWORD: changeme
```

Comment:
I'm not sure. This manifest is just an example so I guess it's fine with `changeme` but of course I can put some random chars in there. If one want to deploy st2 on k8s, they should provide their own `configmap` or even they should use k8s `secret` for sensitive information like passwords.